database/schema.md

```diff
@@ -0,0 +1,113 @@
+# Schema
+
+Below is an explanation of the current database schema. This schema is duplicated across the (currently) two database backends we support: sqlite and postgres.
+
+## Tables
+
+### benchmark
+
+The different types of benchmarks that are run. 
+
+The table stores the name of the benchmark as well as whether it is capable of being run using the stable compiler.  The benchmark name is used as a foreign key in many of the other tables. 
+
+```
+sqlite> select * from benchmark limit 1;
+name        stabilized
+----------  ----------
+helloworld  0   
+```
+
+### artifact
+
+A description of a rustc compiler artifact being benchmarked. 
+
+This description includes:
+* name: usually a commit sha or a tag like "1.51.0" 
```

Comment:
I think it'd be good to note here this is not a "git tag" and is basically free-form text.